pytorch_lightning/loops/base.py

```diff
@@ -314,7 +317,6 @@ def load_state_dict(
         for k, v in self.__dict__.items():
             if isinstance(v, Loop):
                 v.load_state_dict(state_dict.copy(), prefix + k + ".")
-        self.restarting = True
```

Comment:
Why it is being removed? Instead, I would manually reset this value on the loops from the Tuner side. This would be cleaner IMO. @carmocca

Reply:
yeah, I'll be updating this code.